packages/cli/src/templates/project/package-json.ts

```diff
@@ -22,7 +23,7 @@ export const template = `{
     "mocha": "^7.0.1",
     "nyc": "^15.0.0",
     "prettier": "^1.19.1",
-    "typescript": "^3.7.5",
+    "typescript": "^3.9.3",
```

Comment:
Nice, with this change, we could start using `Promise.allSettled`.

Reply:
In that case, maybe it would be better if I stop and change my local branch to use the native method. What do you think guys?